examples/02_model_collaborative_filtering/cornac_bivae_deep_dive.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>I expect the actual ratings are still meaningful with this approach? Do you want to include rating metrics as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
The current implementation doesn't favor rating metrics (i.e. ratings are binarized implicitly). The focus here is still top-*K* recommendation so I think it's better to just keep ranking metrics.

Reply:
got it, makes sense